girder/api/v1/resource.py

```diff
@@ -180,7 +180,7 @@ def _lookUpToken(self, token, parentType, parent):
         raise RestException('Child resource not found: %s(%s)->%s' % (
             parentType, parent.get('name', parent.get('_id')), token))
 
-    def _lookUpPath(self, path, user):
+    def _lookUpPath(self, path, test, user):
```

Comment:
Thanks for the contribution @xerxes455 .

Change this function def to better support backwards compatibility and provide default value for `test`

    def _lookUpPath(self, path, user, test=False):

Can you also add a docstring for this function?